src/components/TheHeader.vue

```diff
@@ -23,6 +23,7 @@
             :data-cy="cyName"
             :title="text"
             :to="page"
+            :style="{margin :'4px'}"
           >
             <v-icon
               :icon="`mdi-${icon}`"
```

Comment:
This is where we should put the margin class so that it goes between the icon and text

Reply:
I opened a new PR with required changes. Please check